vignettes/admiral.Rmd

```diff
@@ -42,8 +42,32 @@ ex <- admiral_ex
 # derive treatment variables
 adsl <- dm %>%
   mutate(TRT01P = ARM, TRT01A = ACTARM) %>%
-  derive_var_trtsdtm(dataset_ex = admiral_ex) %>%
-  derive_var_trtedtm(dataset_ex = admiral_ex) %>%
+  derive_vars_merged_dtm(
+    dataset_add = ex,
+    filter_add = (EXDOSE > 0 |
+                    (EXDOSE == 0 &
+                       str_detect(EXTRT, "PLACEBO"))) &
+      nchar(EXSTDTC) >= 10,
```

Comment:
Do we need this filter for these beginner vignettes?  Just a lot to digest on the intro level.  

If so, maybe we should add in a few Notes about what is happening here??

Reply:
We do talk about the filter parameter later in the Derive Disposition variables  section

Reply:
I have simplified the condition.